colorfield/templates/colorfield/color.html

```diff
@@ -1,9 +1,10 @@
 <input type="text"
  id="{{ id }}"
- class="colorfield_field jscolor {% if class %}{{ class }}{% else %}form-control{% endif %}"
+ class="colorfield_field coloris {{ id }} {% if class %}{{ class }}{% else %}form-control{% endif %}"
  name="{{ name }}"
  value="{% firstof value default '' %}"
  placeholder="{% firstof value default '' %}"
- data-jscolor="{hash:true,width:225,height:150,format:'{{ format }}',required:{% if required %}true{% else %}false{% endif %},paletteCols:4,paletteHeight:28{% if palette %},palette:{{ palette|safe }}{% endif %}}"
+ data-coloris="{{ data_coloris }}"
```

Comment:
I thought more about this, the conf data can be converted to json and escaped at template level `{{ data_coloris|json|escape }}`.

I think the most correct and safe way to pass this json conf would be to use `json_script` template tag, but considering that we know what data will be rendered it would probably over-complicate the data forwarding.

Reply:
There is no `json` template-tag the only one is the `json_script` but it writes it inside a `<script></script>` tag.

Reply:
True, let's use `escape` filter.

Reply:
The right way for doing this is to use `json_script` template tag, that yes, it adds a new script tag, but that's not a problem. Could you try to do it with `json_script`?